### PR TITLE
add cookie AppId specific

### PR DESCRIPTION
**Pull request by alexeylindin**
Current implementation doesn't support rendering multiple banners across pages. 
E.g. Home page has smart-banner for app1, and 3rd level page got app2. If we close smart-banner for app1, we won't render smart-banner app2.

### CODE REVIEW

**Review by GabLeRoux — COMMENTED** [review]
Code looks good to me, but once this is merged for everyone who clicked the `x`, the banner will be back for them since cookie key will change. Using a dynamic cookie key is a good idea 👍 

Should we make it configurable so one could update to this, but still keep older cookie key?

**Comment by alexeylindin** [comment]
Nice question! From my point of view from business perspective it would be nice to remind once more about application. This is the first option. Person will click/close banner and everything will work the same way as right now.

If this is not suitable option I could make additional check for old cookie. However it's not clear in case of multiple application. How could we determine that existing cookie refers to appId1 or appId2? So as the second option I would propose to remove old one if it is found, show banner and place new one.

File: index.js [hunk 1]
```diff
@@ -75,12 +75,15 @@ var SmartBanner = function (options) {
 	// - user is on mobile safari for ios 6 or greater (iOS >= 6 has native support for SmartAppBanner)
 	// - running on standalone mode
 	// - user dismissed banner
+	this.appMeta = mixins[this.type].appMeta;
```
Comment (Seldaek) [inline index.js:78]: This statement breaks on desktop as this.type is undefined.

Comment (Seldaek) [reply index.js:78]: @dy could you please look into this and make a new release?

Comment (alexeylindin) [reply index.js:78]: @Seldaek Hello. I've just tested inside test folder. What's the purpose of rendering smart-banner on desktop?

Comment (Seldaek) [reply index.js:78]: There is no purpose of course but I don't wanna deal with device detection if the library is doing it anyway and does not show the banner when on desktop. It worked fine until now, but this generates an error.

Comment (alexeylindin) [reply index.js:78]: Hmm, I've looked through comments and found options when we should not render anything "device os is not supported". This options is based on device type, but it is placed after my statement. To fix current issue i would suggest to move type check up so when there is no appropriate type we won't construct SmartBanner.

I've tested current 1.5.0 version and see issue you described. My local changes fixes this.

Comment (alexeylindin) [reply index.js:78]: @Seldaek comment this one please https://github.com/kudago/smart-app-banner/pull/101

**Pull request merged**